Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Build the manager binary
+FROM golang:1.10.3 as builder
+
+# Copy in the go src
+WORKDIR /go/src/github.com/platform9/azure-provider
+COPY pkg/    pkg/
+COPY cmd/    cmd/
+COPY vendor/ vendor/
+
+# Build
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o manager github.com/platform9/azure-provider/cmd/manager
+
+# Copy the controller-manager into a thin image
```

Comment:
Is alpine 3.8 possible?

Reply:
yup - it should be. The file was generated using kube-builder and I think it defaulted to ubuntu.